workflow-controller/controller-service/src/main/java/io/hyscale/controller/commands/HyscaleExecCommand.java

```
@@ -0,0 +1,76 @@
+import io.hyscale.controller.builder.K8sAuthConfigBuilder;
+import io.hyscale.controller.util.CommandUtil;
+import io.hyscale.deployer.services.deployer.Deployer;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "exec", description = { "Exec into  specified container" })
+@Component
+public class HyscaleExecCommand implements Callable<Integer> {
+	
+	@Pattern(regexp = ValidationConstants.SERVICE_NAME_REGEX, message = ValidationConstants.INVALID_SERVICE_NAME_MSG)
+	@Option(names = { "-s", "--service" }, required = true, description = "Service name")
+	private String serviceName;
+	
+	@Pattern(regexp = ValidationConstants.APP_NAME_REGEX, message = ValidationConstants.INVALID_APP_NAME_MSG)
+	@Option(names = { "-a", "--app" }, required = true, description = "Application name")
+	private String appName;
+	
+	@Pattern(regexp = ValidationConstants.REPLICA_NAME_REGEX, message = ValidationConstants.INVALID_REPLICA_NAME_MSG)
+	@Option(names = { "-r", "--replica" }, required = true, description = "Replica name")
+	private String replicaName;
+	
+	@Pattern(regexp = ValidationConstants.NAMESPACE_REGEX, message = ValidationConstants.INVALID_NAMESPACE_MSG)
+	@Option(names = { "-n", "--namespace", "-ns" }, required = true, description = "Namespace of the service")
+	private String namespace;
+	
+	
+	 @Autowired
+	 private K8sAuthConfigBuilder authConfigBuilder;
+	 
+	 @Autowired
+	 private Deployer deployer;
+
+	@Override
+	public Integer call() throws Exception {
+		if (!CommandUtil.isInputValid(this)) {
+			return ToolConstants.INVALID_INPUT_ERROR_CODE;
+		}
+		authConfigBuilder.getAuthConfig();
+		K8sConfigFileAuth k8sConfigFileAuth=new K8sConfigFileAuth();
+		k8sConfigFileAuth.setK8sConfigFile(new File(System.getProperty("user.home") + "/.kube/config"));
+		K8sAuthorisation k8sAuthorisation=k8sConfigFileAuth;
+		deployer.exec(k8sAuthorisation,serviceName, appName, namespace, replicaName);
+		return null;
```

This return is used as exit code.
On success return 0.
Else deployer should throw the exception. catch it, log it and rethrow